Add reference

kumarpalsinh25 opened the pull request:
As continuation of [PR - RefDetail enhancement on Space Objects](https://github.com/acterglobal/a3/pull/2440), This PR allow you to directly add reference in similar way we are adding attachments.

https://github.com/user-attachments/assets/ac5313f3-6a99-4103-b5fd-d2f8c1886bec

gnunicorn commented on app/lib/features/tasks/pages/task_list_details_page.dart:
```diff
@@ -162,7 +162,7 @@ class _TaskListPageState extends ConsumerState<TaskListDetailPage> {
     return SingleChildScrollView(
       child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 20.0),
+        padding: const EdgeInsets.symmetric(horizontal: 8.0),
```
what is this about? I didn't see this in the promo video...?

kumarpalsinh25 replied:
Ahh, I was play with TaskList Details padding-spacing improvements. Here with [this commit](https://github.com/acterglobal/a3/pull/2464/commits/de1c7fa4d7f9148702d8dede7acb7b3da647ebc2) I completed improvements. Also sharing screenshot for reference.

![TaskList](https://github.com/user-attachments/assets/631dde4b-7be9-43c5-b64b-8d3381cb4c9a)

gnunicorn commented on app/lib/features/attachments/widgets/attachment_section.dart:
```diff
@@ -187,40 +219,20 @@ class FoundAttachmentSectionWidget extends ConsumerWidget {
           ),
-          child: Text(lang.add),
         ),
+        IconButton(onPressed: onTapAdd, icon: Icon(Icons.add)),
```
shouldn't the ability to this be conditioned by `canEdit`? Because if they can't edit the item, we don't want to show the button...

kumarpalsinh25 replied:
Ya true, let me add it. Good notice!

gnunicorn reviewed (approved):
Generally fine, but two requests.

also wondering if we could have tests for this ...

kumarpalsinh25 commented:
> Generally fine, but two requests.

- Addressed given feedback and added comments over there.

> also wondering if we could have tests for this ...

For attachment section?

gnunicorn commented:
well, generally have tests, @kumarpalsinh25 - e.g. that canEdit is observed etc ...

kumarpalsinh25 commented:
> well, generally have tests, @kumarpalsinh25 - e.g. that canEdit is observed etc ...

Facing some difficulty here, would like to discuss about this on our next call.
